tests_new/integration_tests/modules/s3_datasets/test_s3_tables_filters.py

```diff
@@ -0,0 +1,149 @@
+import logging
+
+from assertpy import assert_that
+import pytest
+
+from integration_tests.modules.s3_datasets.queries import (
+    create_table_data_filter,
+    delete_table_data_filter,
+    list_table_data_filters,
+)
+from integration_tests.errors import GqlError
+
+log = logging.getLogger(__name__)
+
```

Comment:
It is not required yet, but if in the future we implement the shares_filter tests, the COL_INPUT and ROW_INPUT should include sample_data column names